mw-webapp/src/component/table/Table.tsx

```diff
@@ -29,7 +28,7 @@ export const Table = () => {
     columns,
     getCoreRowModel: getCoreRowModel(),
   });
-
+  let keyCounter = 0;
   return (
     <div className={styles.container}>
       <table className={styles.table}>
```

Comment:
why did you change table if you task does not related to it and you did not use tale any where?

Reply:
![image](https://github.com/tritonJS826/masters-way/assets/95077794/a1769c29-d318-4a29-8b9d-de1d7435677b)